pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -242,6 +242,22 @@ func CreateService(name string, service kobject.ServiceConfig, objects []runtime
 	servicePorts := ConfigServicePorts(name, service)
 	svc.Spec.Ports = servicePorts
 
+	// Configure service types
+	for key, value := range service.Annotations {
+		if key == "kompose.service.type" && (value == "NodePort" || value == "nodeport") {
```

Comment:
it would be better to have value test case insensitive.
you can convert it to lowercase `strings.ToLower(value)` and use it in `if` statement